### PR TITLE
Completed css-canvas-medical - Jonathan Chiang

### PR DESCRIPTION
**Pull request by J0N-C**
css-canvas-medical-home-pt1
css-canvas-medical-home-pt2

### CODE REVIEW

File: css-canvas-medical/index.html [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Canvas Medical</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+
+<header>
```
Comment (BrettAlbright) [inline css-canvas-medical/index.html]: This element and its children need to be placed within the `body` tags

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
small update!

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Comment by J0N-C** [comment]
css-canvas-medical-product
css-canvas-medical-implementation
css-canvas-medical-about
css-canvas-medical-blog
css-canvas-medical-contact

**Pull request merged**